packages/module/src/SystemMessageEntry/SystemMessageEntry.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss';
+
+export interface SystemMessageEntryProps {
+  text: string;
+  href: string;
+}
+
+const useStyles = createUseStyles({
+  systemMessageText: {
+    paddingBottom: "var(--pf-v5-global--spacer--md)",
+    textAlign: "center",
+  }
+})
+
+export const SystemMessageEntry: React.FunctionComponent<SystemMessageEntryProps> = (props) => {
+  const classes = useStyles();
+  return (
+    <TextContent>
+      <Text component={TextVariants.small} className={classes.systemMessageText}>
+        {props.text} {props.href}
```

Comment:
I think we could just use {props.children} here. (You would need to add to the props as in [here](https://github.com/patternfly/virtual-assistant/blob/main/packages/module/src/VirtualAssistant/VirtualAssistant.tsx#L54))